### PR TITLE
add gitlab group description

### PR DESCRIPTION
**Pull request by casz**

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java [hunk 1]
```diff
@@ -338,10 +339,14 @@ private PersonalAccessToken getWebHookCredentials(SCMSourceOwner owner) {
             String fullName = gitlabOwner.getFullName();
             String webUrl = gitlabOwner.getWebUrl();
             String avatarUrl = gitlabOwner.getAvatarUrl();
+            String description = null;
+            if (gitlabOwner instanceof GitLabGroup) {
+                description = ((GitLabGroup) gitlabOwner).getDescription();
+            }
             List<Action> result = new ArrayList<>();
             result.add(new ObjectMetadataAction(
                 Util.fixEmpty(fullName),
```
Comment (baymac) [inline src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java]: The display name of GitLab Group is not updated but the description is updated. :/

Comment (casz) [reply src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java]: what? 😭
So you can get some absurd long names 😆

Comment (casz) [reply src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java]: @baymac if user has given the group a display name or description it does not override

Comment (casz) [reply src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java]: This works for groups: 
image

**Comment by casz** [comment]
moved to #40 as there were more fixes to the api builder that simplified the try catch

**Pull request closed without merging**